docs/zulipbot-usage.md

```diff
@@ -42,7 +42,7 @@ followed by the desired labels enclosed within double quotes (`""`).
     `@zulipbot remove` followed by the desired labels enclosed with double quotes
     (`""`).
 
-* **Find unclaimed issues** - Use the [GitHub search
+* **Find unclaimed issues** — Use the [GitHub search
```

Comment:
Not sure why this line was changed?

Reply:
The hyphen became a bit longer :) I did it to make the document more consistent. Please have a look at the [rich diff](https://github.com/zulip/zulip/pull/6637/files/b75b4d278b2b32d81123b880fc4929024673e16d?short_path=990a84d#diff-990a84d2f82c99c0f4d63b27dff2e74f).

Reply:
Got it.